lz4/stream/__init__.py

```diff
@@ -0,0 +1,201 @@
+
+            Raises:
+                Exceptions occuring during decompression.
+
+                ValueError: raised if the source is inconsistent with a finite LZ4
+                    stream block chain.
+                MemoryError: raised if the work output buffer cannot be allocated.
+                OverflowError: raised if the source is too large for being decompressed
+                    in the given context.
+                LZ4StreamError: raised if the call to the LZ4 library fails. This can be
+                    caused by ``decompressed_size`` being too small, or invalid data.
+
+        """
+        return _decompress(self._context, chunk)
+
+    def get_block(self, stream):
+        """ Return the first LZ4 compressed block from ``stream``.
+
+            Args:
+                stream (str, bytes or buffer-compatible object): LZ4 compressed stream.
+
+            Returns:
+                bytes or bytearray: LZ4 compressed data block.
+
+            Raises:
+                Exceptions occuring while getting the first block from ``stream``.
+
+                BufferError: raised if the function cannot return a complete LZ4
+                    compressed block from the stream (i.e. the stream does not hold
+                    a complete block).
+                MemoryError: raised if the output buffer cannot be allocated.
+                OverflowError: raised if the source is too large for being handled by
+                    the given context.
+
+        """
+        return _get_block(self._context, stream)
+
+
+class LZ4StreamCompressor:
+    """ LZ4 stream compressing context.
+
+    """
+    def __init__(self, strategy, buffer_size, mode="default", acceleration=True, compression_level=9,
+                 return_bytearray=False, store_comp_size=4, dictionary=""):
+        """ Instantiates and initializes a LZ4 stream compression context.
+
+            Args:
+                strategy (str): Buffer management strategy. Can be: ``double_buffer``.
+                buffer_size (int): Base size of the buffer(s) used internally for stream
+                    compression/decompression.
+                    In the ``double_buffer`` strategy case, this is the size of each
+                    buffer of the double-buffer.
+
+            Keyword Args:
+                mode (str): If ``default`` or unspecified use the default LZ4
+                    compression mode. Set to ``fast`` to use the fast compression
+                    LZ4 mode at the expense of compression. Set to
+                    ``high_compression`` to use the LZ4 high-compression mode at
+                    the expense of speed.
+                acceleration (bool): When mode is set to ``fast`` this argument
+                    specifies the acceleration. The larger the acceleration, the
+                    faster the but the lower the compression. The default
+                    compression corresponds to a value of ``True``.
```

Comment:
What is the range of values for `acceleration`? What does a value of `True` actually mean? I presume this is supposed to be a numeric value, in which case I think the default should be a number, not `True`.